package.json

```diff
@@ -1,18 +1,24 @@
 {
-  "name": "chernivtsi-transport",
+  "name": "gps",
   "version": "0.1.0",
   "private": true,
-  "workspaces": [
```

Comment:
`workspaces` was added lately. Please, keep them. They're required to work with Lerna.